### DISCUSSION TITLE
Question about Symfony CI tests

### DISCUSSION THREAD

**Discussion by gnito-org** [symfony/symfony#43064] [Q&A]
Please refer to https://github.com/symfony/symfony/pull/43018 and https://github.com/symfony/symfony/pull/43053.

Why do you run low deps 4.4 tests against a pull request that specifically targets 5.4? The code of the pull request will never run in a Symfony 4.4 environment, so why test against it?

Is it on purpose or is it just a limitation of the CI system?

**Comment by mbabker** [comment]
https://symfony.com/doc/current/contributing/code/pull_requests.html#automated-tests explains the purpose of the build configs.
